.github/workflows/release.yml

```diff
@@ -144,6 +127,7 @@ jobs:
           echo "BUILD_NAME=${build_name}" >> $GITHUB_ENV 
           echo "BUILD_NUMBER=${build_nbr}" >> $GITHUB_ENV 
           echo "BUILD_VERSION=${version}" >> $GITHUB_ENV 
+     
```

Comment:
You're adding a new unnecessary line here

Reply:
done